heap/heap.cpp

```diff
@@ -0,0 +1,33 @@
+#include "heap.h"
+#include <vector>
+
+void heap::sift_up(int i){
+    if (i==0){
```

Comment:
```suggestion
    if (i==0) {
```
в общем, везде поправь такие места